tests/run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+if [[ "$NO_COLOURS" != "1" ]]; then
+    RESTORE='\033[0m'
+    RED='\033[00;31m' GREEN='\033[00;32m' YELLOW='\033[00;33m' BLUE='\033[00;34m' PURPLE='\033[00;35m' CYAN='\033[00;36m'
+fi
+
+TESTS_SUCCEED=0
+TESTS_FAILED=0
+TESTS_SKIPPED=0
+
+TESTS_NAMES=$1
+
+if [[ "$TESTS_NAMES" != "" ]]; then
+    TESTS="$TESTS_NAMES"
+    SHOW_OUTPUT=1
+else
+    TESTS=`ls -d ./tests/*/run.sh | cut -f3 -d'/'`
+fi
+
+echo -e "${BLUE}=====================PostDock tests======================${RESTORE}"
+
+for TEST in $TESTS; do
+    echo -n ">>> Running test $TEST":
+    if [[ "$DEBUG" == "1"  ]]; then
+        echo -e " ${BLUE}>>>>>>${RESTORE}"
+        ./tests/$TEST/run.sh
+        TEST_EXIT_CODE=$?
+        echo -ne "${BLUE}<<<<<<${RESTORE}"
+    else
+        TEST_OUTPUT=$(./tests/$TEST/run.sh 2>&1)
+        TEST_EXIT_CODE=$?
+    fi
+    if [[ "$NO_CLEANUP" != "1" ]]; then
+        docker-compose down -v
```

Comment:
Maybe we should suppress output from this command?

Reply:
need to move on block down